article/_posts/2022-10-07-ExplainableDisentangledRepresentations.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: review
+title: "Improving Explainability of Disentangled Representations using Multipath-Attribution Mappings"
+tags: deep-learning representation-learning autoencoder VAE medical
+author: "Nathan Painchaud"
+cite:
+    authors: "Lukas Klein, João B. S. Carvalho, Mennatallah El-Assady, Paolo Penna, Joachim M. Buhmann, Paul F Jaeger"
+    title:   "Improving Explainability of Disentangled Representations using Multipath-Attribution Mappings"
+    venue:   "MIDL 2022"
+pdf: "https://openreview.net/pdf?id=3uQ2Z0MhnoE"
+---
+
+
+# Highlights
+- Leverage existing SOTA disentangled representation method ($$\beta$$-TCVAE) to learn interpretable representation for
+downstream tasks;
+- Introduce a new multi-path attribution mapping to explain what guided a downstream networks' prediction (the
+attribution w.r.t. the input), but also why it had an impact (the attribution w.r.t. the interpretable representation);
+- Method helps to extract causal relations, as well as improve robustness to distribution shifts by detecting
+"shortcuts" learned by models.
+
+
+# Introduction
+The authors argue that current approaches in interpretable deep learning are too prone to interpret correlation as
+causation. Ultimately, wrongly identifying the underlying causal relations also affects generalization capabilities,
+since it makes models more vulnerable to domain shifts, by correlating non-causal factors of variation to the prediction
+target.
+
+Thus, the authors' motivation is to provide a better framework to interpret correlations in the data, while leaving the
+task to discriminate between correlation and causation to domain experts.
+
+
+# Methods
+
+The overall method is summarized in the figure below, and explained in more details in the following subsections:
+
+![](/article/images/ExplainableDisentangledRepresentations/figure1.jpg)
+
+## Autoencoder
+To learn human-interpretable factors of variation to be used later in the attribution mechanism, the authors use the
+$$\beta$$-TCVAE[^1]. Very briefly, $$\beta$$-TCVAE is a method that decomposes the KL divergence in VAEs into 3 terms,
+notably the total correlation of the aggregate posterior:
+
+$$
+KL(q(z)~\|~\prod_{j=1}^{d} q(z_{j})), \quad \text{with}~d~\text{the number of latent dimensions}
```

Comment:
The tilde are only meant to introduce a single space. The spaces are not _that_ noticeable, because the formula is still quite dense, but it's when they're missing that the lack of spaces really becomes noticeable :wink: 

![image](https://user-images.githubusercontent.com/23144457/199339398-c30d3f37-cd07-4fa9-8f32-99278ada3549.png)

Reply:
I know that the tildes are intended to introduce a single space (AKA whitespace :-)). OK, might be the preview of GitHub then.

![ExplainableDisentangledRepresentations_tildes](https://user-images.githubusercontent.com/5576557/199342213-4ced1f17-1fba-406d-ab40-78e0aee7b6c3.png)

Resolving.